docs/governance/rfc/005-python-implementation.md

```diff
@@ -0,0 +1,127 @@
+# 005: Python Implementation
+
+I propose an official Python implementation of Connect, based on the prototype at https://github.com/firetiger-oss/connect-python/.
```

Comment:
This is going to end up in https://connectrpc.com/docs/governance, so we should clean this up:

- Make this "This RPC proposes" not "I propose an"
- No link needed to the protoype, since this won't exist if we merge this RFC